voting_schemes/electionguard/python-wrapper/src/bulletin_board/electionguard/bulletin_board.py

```diff
@@ -14,7 +14,7 @@
 from electionguard.type import GUARDIAN_ID
 from electionguard.utils import get_optional
 
-from .common import Content, Context, ElectionStep, Wrapper
+from .common import Content, Context, ElectionStep, Wrapper, async_wrap
```

Comment:
I think this import wasn't necessary
```suggestion
from .common import Content, Context, ElectionStep, Wrapper
```